README.md

```diff
@@ -85,8 +85,7 @@ Please install the following dependencies before building Facebook Thrift:
 
 This will create:
 
-* `thrift/bin/thrift1`: The Thrift compiler binary to generate client and
-  server code.
+* `thrift/bin/thrift1`: The Thrift compiler binary to generate client and server code.
```

Comment:
Thanks for the PR but please avoid whitespace changes.

Reply:
Okay!